sample/src/main/java/com/commit451/coiltransformations/sample/App.kt

```diff
@@ -6,21 +6,18 @@ import android.app.Application
 import android.util.Log
 import coil.Coil
 import coil.ImageLoader
+import coil.memory.MemoryCache
 import coil.util.DebugLogger
 
 class App : Application() {
 
     override fun onCreate() {
         super.onCreate()
-        Coil.setImageLoader(::buildDefaultImageLoader)
-    }
-
-    private fun buildDefaultImageLoader(): ImageLoader {
-        return ImageLoader.Builder(this)
-                .logger(DebugLogger(level = Log.DEBUG))
-                .availableMemoryPercentage(0.5)
-                .bitmapPoolPercentage(0.5)
-                .crossfade(true)
-                .build()
+        val imageLoader = ImageLoader.Builder(this)
```

Comment:
I'd recommend implementing `ImageLoaderFactory` instead of using `setImageLoader` since `ImageLoaderFactory` creates the `ImageLoader` lazily.